sphinx_docs/source/alcf.rst

```diff
@@ -108,3 +111,42 @@ A script that can be used to chain jobs with PBS is:
 .. literalinclude:: ../../job_scripts/polaris/chainqsub.sh
    :caption: ``chainqsub.sh``
 
+
+Installing Python
+=================
+
+The most recommended way to install python is to create a virtual environment
+on the top of the conda-based environment provided by the module conda, and install all the extra
+required modules on this virtual environment. Although is very tempting
+to clone the whole base environment and fully customize the installed conda packages, some
+modules like ``mpi4py`` may require access to the MPICH libraries that are tailored to be
+used only within the conda-base environment provided by the conda module.
+
+To create the virtual environment:
+
+.. prompt:: bash
+
+   module use /soft/modulefiles
+   module load conda 
+   conda activate
+   VENV_DIR="venvs/polaris"
+   mkdir -p "${VENV_DIR}"
+   python -m venv "${VENV_DIR}" --system-site-packages
+   source "${VENV_DIR}/bin/activate"
```

Comment:
We should mention these commands come from https://docs.alcf.anl.gov/polaris/data-science-workflows/python/, so we know where to look if this stops working

Reply:
I mentioned it!